feat: openemr fix #6631,#6632 twig datetime filters,multiple template path directories

adunsulag opened the pull request:
Fixes #6631 
Fixes #6632 

Add datetime filter for twig to respect OpenEMR global times.

Add multiple search paths to the twig template container.

robertdown commented:
Since we have a default method for handling dates, I recommend we simply overwrite the built in `date` filter. That way we save a few extra keystrokes and don't go down the path of prepending redundant function names with `oe` (a practice I generally really dislike).

adunsulag commented:
@robertdown Figured out a way to configure the default date format for twig instead of override the date filter function.  It accomplishes the same purpose and allows us to drop the custom oeDate function I added.  I believe this addresses your concerns.

robertdown commented:
@adunsulag good opportunity to look at the use of namespaces for our templates. We are probably a small PR away from allowing modules to inject a custom path. So if you wanted to redirect the Medical Record Dashboard to your module's own custom solution, adding that path to the namespace would allow that.
